### ISSUE TITLE
[Bug] Single post download fails when output directory omitted

### ISSUE THREAD

**Issue by fireundubh** [althonos/InstaLooter#72]
Downloading a single post from a URL fails with nonsensical error (to the user) when the output directory argument is omitted.

Example from the readme:
> Download a single post from an url in the current directory
> `$ instaLooter post "https://www.instagram.com/p/BFB6znLg5s1/" .`

Omit the output directory argument:
> `$ instaLooter post "https://www.instagram.com/p/BFB6znLg5s1/"`

Returns this error:
```
Traceback (most recent call last):
  File "c:\dev\python27\lib\runpy.py", line 174, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "c:\dev\python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "D:\dev\Python27\Scripts\instaLooter.exe\__main__.py", line 9, in <module>
  File "c:\dev\python27\lib\site-packages\instaLooter\utils.py", line 76, in new_func
    result = func(*args, **kwargs)
  File "c:\dev\python27\lib\site-packages\instaLooter\cli.py", line 123, in main
    template=args['--template']
  File "c:\dev\python27\lib\site-packages\instaLooter\core.py", line 102, in __init__
    os.makedirs(directory)
  File "c:\dev\python27\lib\os.py", line 150, in makedirs
    makedirs(head, mode)
  File "c:\dev\python27\lib\os.py", line 150, in makedirs
    makedirs(head, mode)
  File "c:\dev\python27\lib\os.py", line 150, in makedirs
    makedirs(head, mode)
  File "c:\dev\python27\lib\os.py", line 157, in makedirs
    mkdir(name, mode)
WindowsError: [Error 123] The filename, directory name, or volume label syntax is incorrect: 'https:'
```

**Comment by althonos** [comment]
Hi, this is a docopt limitation that was caused by the docopt parser recognizing `post` and `hashtag` commands as profile names, and whatever followed as a directory.

I manually overrode this issue, so now `$ instaLooter post "https://www.instagram.com/p/BFB6znLg5s1/"` will print the usage instead (actually the message was kinda Windows-specific, on Linux it created a stupid directory tree and then stopped immediately because the user named `post` has a private account).

**Comment by althonos** [comment]
Fixed in `v0.9.9`.
